test/integration/consul-container/libs/service/helpers.go

```diff
@@ -121,3 +133,48 @@ func CreateAndRegisterStaticClientSidecar(
 
 	return clientConnectProxy, nil
 }
+
+func GetEnvoyConfigDump(port int, filter string) (string, error) {
```

Comment:
I think this function can be replaced by calling GetEnvoyOutput(port, "config_dump", filters)